app/components/c-collapse-group.md

```diff
@@ -0,0 +1,27 @@
+```
+<template>
+  
+  <c-collapse-group>
+  
+      <c-collapse
+        v-for="item in collapsible"
+        :key="item.id"
+        :title="item.title">
+        {{ item.content }}
+      </c-collapse>
+      
+  </c-collapse-group>
+
+</template>
+
+<script>
+  export default {
+    name: 'collapse',
+    data() {
+      return {
+        collapsible: this.$store.getters['product/collapsible']
```

Comment:
- [ ] I think we should move this mock data somewhere else. When the real data comes, it probably will be handled completely different. I would recommend to create a styleguide module in the `/app/styleguide` folder and inject the module inside `/app/setup/styleguidist.js` with "Dynamic module registration": https://vuex.vuejs.org/en/modules.html#dynamic-module-registration

This way mock data will also not end up in live code. Note that this is possibly a rare exception, since I don't think we will have such an array we can loop trough for the accordion in production. In most cases the mock data will represent a real implementation. Still we need to keep an eye on the build process to not include mock data in the build.

Reply:
As discussed I'm gonna change this in a separate branch.